src/patternfly/components/DualListSelector/dual-list-selector.scss

```
@@ -57,8 +57,13 @@ $pf-v6-c-dual-list-selector__item--MaxNesting: 10;
   // List item
   --#{$dual-list-selector}__list-item-row--FontSize: var(--pf-t--global--font--size--sm);
   --#{$dual-list-selector}__list-item-row--BackgroundColor:  var(--pf-t--global--background--color--action--plain--default);
+  --#{$dual-list-selector}__list-item-row--BorderWidth: var(--pf-t--global--border--width--regular);
+  --#{$dual-list-selector}__list-item-row--BorderColor: transparent;
```

Can we update this so the border-width is what enables the border?

```suggestion
  --#{$dual-list-selector}__list-item-row--BorderWidth: 0;
  --#{$dual-list-selector}__list-item-row--BorderColor: var(--pf-t--global--border--color--high-contrast);
```